pkg/trivy/scanner.go

```diff
@@ -315,10 +315,11 @@ func (s *scanner) getPodSpecForClientServerMode(spec corev1.PodSpec, credentials
 		}
 
 		containers = append(containers, corev1.Container{
-			Name:            container.Name,
-			Image:           s.config.GetTrivyImageRef(),
-			ImagePullPolicy: corev1.PullIfNotPresent,
-			Env:             env,
+			Name:                     container.Name,
+			Image:                    s.config.GetTrivyImageRef(),
+			ImagePullPolicy:          corev1.PullIfNotPresent,
+			TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
```

Comment:
We could not see the error logs coming from pod logs in case of failure. (In ClientServer mode)